packages/amplication-git-service/src/providers/github.service.ts

```diff
@@ -227,9 +228,15 @@ export class GithubService implements IGitClient {
 
     const files = Object.fromEntries(
       modules.map(module => {
+        // ignored file
         if (amplicationIgnoreManger.isIgnored(module.path)) {
           return [join(AMPLICATION_IGNORED_FOLDER, module.path), module.code];
         }
+        // Deleted file
+        if (module.code === null) {
+          return [module.path, module.code];
+        }
+        // Regex ignored file
```

Comment:
I think here the comments actually make sense, as they help the developer understand the flow. So this kind of comments we can keep.